deploy/11_set_dependencies_on_specific_group_strategy.ts

```diff
@@ -0,0 +1,29 @@
+import { DeployFunction } from "@celo/staked-celo-hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { executeAndWait } from "../lib/deploy-utils";
+import { SpecificGroupStrategy } from "../typechain-types/SpecificGroupStrategy";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const account = await hre.deployments.get("Account");
+  const groupHealth = await hre.deployments.get("GroupHealth");
+  const defaultStrategy = await hre.deployments.get("DefaultStrategy");
+  const specificGroupStrategy: SpecificGroupStrategy = await hre.ethers.getContract(
+    "SpecificGroupStrategy"
+  );
+  const multisig = await hre.deployments.get("MultiSig");
+
+  if ((await specificGroupStrategy.callStatic.owner()) !== multisig.address) {
```

Comment:
Should we also add an else statement (here and for any other similar script), indicating the deployer to submit a multisig proposal if owner is already set?